src/translations/views/TranslationsProductVariants.tsx

```diff
@@ -116,7 +122,11 @@ const TranslationsProductVariants: React.FC<TranslationsProductVariantsProps> =
       onDiscard={onDiscard}
       onSubmit={handleSubmit}
       onAttributeValueSubmit={handleAttributeValueSubmit}
-      data={translation?.__typename === "ProductVariantTranslatableContent" ? translation : null}
+      data={
+        translation?.__typename === "ProductVariantTranslatableContent"
+          ? (translation as ProductVariantTranslationFragment)
+          : (null as unknown as ProductVariantTranslationFragment)
```

Comment:
issue: `as unknown as ...` is a big no-no in my opinion, since we're basically lying to TypeScript 🙈 

maybe we can just make `data` nullable in that component?